### PR TITLE
Fixed a couple typos in error messages.

### PR DESCRIPTION
**Pull request by gershwinlabs**
Spotted a couple typos in the Exception messages.  No other changes.

### CODE REVIEW

**Review by djhoese — APPROVED** [review]
Thanks! I'll wait for one of the other maintainers to merge this, but it looks good to me.

**Review by pnuu — APPROVED** [review]
Looks good to me, merging.

**Pull request merged**
